src/main/java/fr/noop/subtitle/model/SubtitleWriterWithInputFrameRate.java

```diff
@@ -0,0 +1,16 @@
+/*
+ *  This file is part of the noOp organization .
+ *
+ *  (c) Cyrille Lebeaupin <clebeaupin@noop.fr>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ */
+
+ package fr.noop.subtitle.model;
+
+
+ public interface SubtitleWriterWithInputFrameRate extends SubtitleWriter {
+     public void setInputFrameRate(String frameRate);
+ }
```

Comment:
j'ai loupé ça, attention à l'indentation et la dernière ligne vide :
```suggestion
package fr.noop.subtitle.model;


public interface SubtitleWriterWithInputFrameRate extends SubtitleWriter {
    public void setInputFrameRate(String frameRate);
}

```